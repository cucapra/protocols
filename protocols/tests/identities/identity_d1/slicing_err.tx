// ARGS: --verilog identities/identity_d1/identity_d1.v --protocol identities/identity_d1/identity_d1.prot
// RETURN: 101
slicing_err(1, 1);