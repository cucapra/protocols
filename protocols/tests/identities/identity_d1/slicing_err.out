error: The two expressions did not evaluate to the same value
   ┌─ identities/identity_d1/identity_d1.prot:34:13
   │
34 │   assert_eq(s[31:15], DUT.s[31:16]);
   │             ^^^^^^^^^^^^^^^^^^^^^^ LHS Value: ValueOwned(00000000000000000), RHS Value: ValueOwned(0000000000000000)

